modules/cluster-management/src/main/java/org/apache/ignite/internal/cluster/management/raft/RocksDbClusterStateStorage.java

```diff
@@ -86,6 +87,13 @@ public RocksDbClusterStateStorage(Path dbPath, String nodeName) {
 
     @Override
     public CompletableFuture<Void> start() {
+        // delete existing data, relying on log playback
+        try {
+            RocksDB.destroyDB(dbPath.toString(), options);
```

Comment:
Judging by the problem, I didn’t notice any of the proposed solutions, or am I looking in the wrong place?

Reply:
This is a solution #1 from the JIRA that I proposed, everything seems fine

Reply:
We discussed it personally with Ivan, and it became clearer.